superset/bin/run_superset_cli.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+if __name__ == '__main__':
```

Comment:
why is this file necessary?

Reply:
bc we need an entry point to run all the available cli commands of **superset** and **fab**
you must run the cli.py file in a command line mode right? 
otherwise you want be able to run it they way it was designed

we already had a bin folder with a entry point similar to that one but lately it was remove probably bc of the inclusion of the` app_context` so I reimplemented by using `with app.app_context():`